packages/caip/src/adapters/mtpelerin/index.ts

```diff
@@ -20,6 +22,7 @@ import {
 const MtPelerinSymbolToAssetIds: Record<string, AssetId[]> = {
   ETH: [ethAssetId, optimismAssetId],
   MATIC: [polygonAssetId],
+  XDAI: [gnosisAssetId],
   BTC: [btcAssetId],
   DAI: ['eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f'],
   USDT: [
```

Comment:
<i class='graphite__hidden'>[Re: lines 75 to 75]<br/></i>

remove me?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/shapeshift/web/4585?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
https://github.com/shapeshift/web/commit/1d8620b1d9723ed0bbc0f7461bf60a7c490b42e4